src/backend/api/handlers/helpers/track_call.py

```diff
@@ -13,7 +13,7 @@ def track_call_after_response(
     Schedules a callback to Google Analytics to track an API call.
     """
     # Save |auth_owner_id| whiel we stil have access to the flask request context.
-    auth_owner_id = g.api_auth_access.owner.id() if g.api_auth_access.owner else None
+    auth_owner_id = g.auth_owner_id if g.auth_owner_id else None
```

Comment:
@fangeugene This bit as well - passing just the ID we need here as opposed to the full object in `g` so we can support other auth mechanisms but still get the logging

Reply:
Makes sense. Unrelated to this particular PR, but I'm wondering if we should wrap the `g` object with some sort of API so it's easier to know what we're setting. Right now it's not clear what we expect to be available in `g` without digging through a lot of code.

Reply:
I filed a tracking issue for this, because this is a good thought, and I don't want to lose sight of it. I've got an idea of how we might go about designing this as well. For now - I wanna try to work on some more pressing issues we have in the codebase. #3386